### ISSUE TITLE
oqenseas.com

### ISSUE THREAD

**Issue by ryder23** [MetaMask/eth-phishing-detect#6073]
Me and a friend are just trying to make a page for memes and NFT content. 

We aren't sure why it's block and would appreciate it if it could get unblocked. #6045

**Comment by devenblake** [comment]
![Looks like this domain has not been registered yet: oqensea.com](https://user-images.githubusercontent.com/26193059/144544060-2bdc27d4-2436-4c6e-92e6-2bcdb23985e5.png)

In the original filed issue you mentioned \#6052 which is a duplicate of #6043, an issue filed by another user *prior* to its registration, so you should know that unregistered domains aren't allowlisted.

**Comment by ryder23** [comment]
I’m sorry I misspelled the domain. It’s oqenseas.com

>

**Comment by devenblake** [comment]
This is an obvious phishing domain.
